pkgs/development/python-modules/gin-config/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, enum34
+, absl-py
+, tensorflow
+, withTensorflow ? true
+}:
+
+buildPythonPackage rec {
+  pname = "gin-config";
+  version = "0.1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zqzhg8nqdqjx0sjrsrs9bpyiaqqlyihqhhy2ijrpp62x9rjllga";
+
+  };
+
+  propogatedBuildInputs = [ six enum34 ] + lib.optional withTensorflow tensorflow;
```

Comment:
this should be `++`.
`lib.optional` does basically the following: `cond: a: if cond then [ a ] else []`.

Reply:
oops thanks! amateur mistake :P